lib/components/Input/base/index.tsx

```diff
@@ -131,36 +158,35 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
                                 statusIconColors[status],
                             )}
                         >
-                            {rightIcon}
+                            {status === "neutral" ? (
+                                <>{rightIcon}</>
+                            ) : (
+                                <>{statusIcon[status]}</>
+                            )}
                         </span>
                     )}
+
                     {triggerActionIcon && <>{triggerActionIcon}</>}
                 </div>
-                <div className="message__container">
-                    {leftStatusMessage && (
-                        <CaptionText
-                            className={clsx(
-                                "message__container__text",
-                                `message__container__text__status--${status}`,
-                                disabled &&
-                                    `message__container__text__disabled`,
-                            )}
-                        >
-                            {leftStatusMessage}
-                        </CaptionText>
-                    )}
-                    {rightStatusMessage && (
-                        <CaptionText
-                            className={clsx(
-                                "self-end",
-                                "message__container__text",
-                                `message__container__text__status--${status}`,
-                            )}
-                        >
-                            {rightStatusMessage}
-                        </CaptionText>
-                    )}
-                </div>
+                {(message || characterCounter) && (
+                    <div
+                        className={clsx(
+                            "message__container",
+                            `message__container--${inputSize}`,
+                            `message__container__status--${status}`,
+                            disabled && `message__container__disabled`,
+                        )}
+                    >
+                        <span className="message__container__text">
+                            {message}
+                        </span>
+                        {characterCounter && maxLength && (
+                            <span className="message__container__text">
+                                {inputValue.length}/{maxLength}
+                            </span>
+                        )}
+                    </div>
+                )}
```

Comment:
**Quick question:** So even if let's say we don't have `message` we still render the UI? Is it because we need it for the UI consistency?